labcodes/confetti-storybook
# CSYS-210: Migrate Tags

**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-210

**Staging app:** https://migrate-tags--confetti-storybook.netlify.app/

**How to test:**
Open the staging app, then check if if the story of `SimpleTag`, `DropdownTag`, `RemovableTag` and `TogglableTag` work as expected.

**Description of your solution:**
This PR adds the stories of Tag components.
